config/nginx-default.conf

```diff
@@ -0,0 +1,28 @@
+server {
+    server_name localhost;
+    listen 80;
+    root /srv/annotations/web;
+
+    location / {
+        try_files $uri /app_dev.php$is_args$args;
+    }
+
+    location ~ ^/app_dev\.php(/|$) {
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $request_filename;
+        fastcgi_param ENVIRONMENT_NAME dev;
+        fastcgi_intercept_errors on;
+        fastcgi_pass php:9000;
```

Comment:
communicating with `php-fpm` over its standard port rather than via the filesystem